tests/buildsystem/schemas/test_utils.py

```diff
@@ -0,0 +1,22 @@
+import os
+import pytest
+import uuid
+
+from buildtest.buildsystem.schemas.utils import load_schema, load_recipe
+
+root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+
+@pytest.mark.xfail(
+    reason="Invalid File Extension for loading schema", raises=SystemExit
+)
+def test_load_schema_invalid_ext():
+
+    # invalid file extension should fail
+    load_schema(os.path.join(root, "README.rst"))
```

Comment:
One thing we will need to think about for another PR is how to handle skipping over yaml files that are discovers that aren’t intended for buildtest.

Reply:
indeed good discussion for another PR :)